src/core/core.tooltip.js

```diff
@@ -87,6 +87,9 @@ module.exports = function(Chart) {
 					backgroundColor: view.backgroundColor
 				};
 			},
+			textLabelColor: function(tooltipItem, chart) {
+				return chart.tooltip._options.bodyFontColor;
```

Comment:
@etimberg  can you please elaborate on this one. I'm not sure i understood that.